assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,391 @@
+form {
+  text-align: center;
+  font-family: $dp-font-family-proximanova;
+
+  .rgpd {
+    margin: 30px 0 10px;
+
+    p {
```

Comment:
i'm not sure if it's safe to use p to declare classes, i think this will lead into a future bug.